rogue-thi-app/pages/calendar.js

```diff
@@ -178,25 +180,43 @@ export default function Calendar () {
             <ListGroup variant="flush">
               <ReactPlaceholder type="text" rows={10} ready={events}>
                 {events && events.length === 0 && (
-                  <ListGroup.Item className={styles.item}>
-                    Es sind derzeit keine Veranstaltungstermine verfügbar.
-                  </ListGroup.Item>
+                    <ListGroup.Item className={styles.item}>
+                      Es sind derzeit keine Veranstaltungstermine verfügbar.
+                    </ListGroup.Item>
                 )}
-                {events && events.map((item, idx) =>
-                  <ListGroup.Item key={idx} className={styles.item}>
-                    <div className={styles.left}>
-                      {!item.url && item.title}
-                      {item.url && (
-                        <a href={item.url} className={styles.eventUrl} target="_blank" rel="noreferrer">
-                          {item.title}
-                          {' '}
-                          <FontAwesomeIcon icon={faExternalLinkAlt} />
-                        </a>
-                      )}
-                      <div className={styles.details}>
-                        {item.organizer} <br />
-                        {item.begin && formatFriendlyDateTimeRange(item.begin, item.end)}
-                      </div>
+                {events && events.map((item, idx) => {
+                  const club = clubs.find(club => club.club === item.organizer)
+                  return <ListGroup.Item key={idx} className={styles.item}>
+                        <div className={styles.left}>
+                          {!item.url && item.title}
+                          {item.url && (
+                              <a href={item.url} className={styles.eventUrl} target="_blank" rel="noreferrer">
+                                {item.title}
+                                {' '}
+                                <FontAwesomeIcon icon={faExternalLinkAlt}/>
+                              </a>
+                          )}
+                          <div className={styles.details}>
+                        <span>
+                          { club != null &&
+                              <>
+                                <a href={club.website} className={styles.eventUrl} target="_blank" rel="noreferrer">
+                                  {item.organizer}<FontAwesomeIcon icon={faExternalLink} fixedWidth/>
+                                </a>
+                                <a href={club.instagram} className={styles.eventUrl} target="_blank" rel="noreferrer">
+                                  <FontAwesomeIcon icon={faInstagram} fixedWidth/>
+                                </a>
```

Comment:
not all clubs have instagram, we need to check if the url is not falsy before rendering it.